doozer

```diff
@@ -851,6 +851,8 @@ def images_build_image(runtime, odcs, repo_type, repo, push_to_defaults, push_to
     failed = [name for name, r in results if not r]
     if failed:
         runtime.logger.error("\n".join(["Build/push failures:"] + sorted(failed)))
+        if len(runtime.missing_pkgs) != 0:
+            runtime.logger.error("Missing pkgs are: \n".join(str(s) for s in runtime.missing_pkgs))
```

Comment:
```suggestion
            runtime.logger.error("Missing pkgs are: \n{}".format("\n".join(runtime.missing_pkgs))
```
Your code was joining on `Missing pkgs are:` which it would've printed out one for each package.
Also, the set is already a set of strings, no need to iterate and convert to string,